src/regridder.jl

```diff
@@ -0,0 +1,19 @@
+
+regridder(grid1, grid2) = GeometryOps.area_of_intersection_operator(grid1, grid2)
+
+
+regridder!(regridder::AbstractMatrix, grid1, grid2) = GeometryOps.area_of_intersection_operator!(regridder, grid1, grid2)
```

Comment:
IMO we should implement these directly here.  Could just copy the code directly, or I'm working on a colab notebook with a better version that I'll send you in a bit.

Reply:
I'll copy the code for now then if your algorithm from https://github.com/JuliaGeo/GeometryOps.jl/pull/283 isn't supposed to land in GeometryOps anyway!